interfaces/functions.yaml

```diff
@@ -1325,8 +1325,9 @@ igraph_is_clique:
 
 igraph_cliques:
     PARAMS: |-
-        GRAPH graph, OUT VERTEX_INDICES_LIST res, INTEGER min_size=0,
-        INTEGER max_size=0
+        GRAPH graph, OUT VERTEX_INDICES_LIST res, 
+        INTEGER min_size=UNLIMITED, INTEGER max_size=UNLIMITED,
+        INTEGER max_results=UNLIMITED
```

Comment:
@ntamas Do we have facilities in Stimulus to indicate, within this repo, that the C value for `UNLIMITED` is `IGRAPH_UNLIMITED`?

Reply:
No, but why would we need that? C does not have default values.